### DISCUSSION TITLE
有没有那种可以在家里工作的方法呀，太社恐完全出不去门。

### DISCUSSION THREAD

**Discussion by justsmile3** [share121/inter-knot#2481] [其他]
问问大家，有没有什么适合在家工作的

**Comment by cza2019** [comment]
在绳网接委托，代肝绝区零

**Comment by 00zzxx** [comment]
好问题，视频运营兼职？我听说那种是可以足不出户的<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by BHGD-031212** [comment]
画插画？<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by jostar9527** [comment]
我个人觉得那些比较适合在家工作的工作较为符合楼主的要求（bushi）<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by zhixuan45** [comment]
美术/计算机/代肝这些，我记得外网好像有个代码外包平台，美术就国内有米画师，画加这些，国外pixiv，推

**Comment by justsmile3** [comment]
谢谢大家的建议
